code-challenges.js

```diff
@@ -3,44 +3,102 @@
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 // Use the test variables provided.
 
-var num1 = 15
-// Expected output: "15 is divisible by three"
 
-var num2 = 0
-// Expected output: "0 is divisible by three"
+// const divisible = (num) => {
+//     if(num % 3 == 0){
+//         return `${num} is divisible by three`
+//     }else{
+//         return `${num} is not divisible by three`
+//     }
+// }
 
-var num3 = -7
-// Expected output: "-7 is not divisble by three"
+// var num1 = 15
+// // Expected output: "15 is divisible by three"
 
+// var num2 = 0
+// // Expected output: "0 is divisible by three"
 
+// var num3 = -7
+// // Expected output: "-7 is not divisble by three"
+
+// console.log(divisible(num3))
```

Comment:
Looks good! But be sure to keep your variables ABOVE your function, as this is considered HOISTING and is not a very good move. Addtionally, lets make sure you keep your code uncommented and each variable tested with a console log. You can comment out the console logs, but leave the code there :)